### PR TITLE
Fix/v6 dropping TxVariants prop for Cards

### PR DESCRIPTION
**Pull request by sponglord**
## Summary
Removing the `static` property `txVariants` for Card related components.
Rationalising how `type` and `brand` are passed down from the various `Card` components to their `CardInputs` and on to the `SecuredFieldProvider` (covers `Card`, `StoredCard`, `CustomCard` & `Bancontact`)

## Tested scenarios
All card related e2e tests pass

### CODE REVIEW

File: packages/lib/src/components/Ach/components/AchInput/AchInput.tsx [hunk 1]
```diff
@@ -204,6 +204,7 @@ export default AchInput;
 
 const extractPropsForSFP = (props: ACHInputProps) => {
     return {
+        allowedDOMAccess: props.allowedDOMAccess,
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/Ach/components/AchInput/AchInput.tsx:207]: Looks like the `allowedDOMAccess` got added again to the codebase. Is it intentional?

File: packages/lib/src/components/Card/Card.tsx [hunk 2]
```diff
@@ -66,7 +65,7 @@ export class CardElement extends UIElement<CardElementProps> {
             // billingAddressRequired only available for non-stored cards
             billingAddressRequired: props.storedPaymentMethodId ? false : props.billingAddressRequired,
             // ...(props.brands && !props.groupTypes && { groupTypes: props.brands }),
-            type: props.type === 'scheme' ? 'card' : props.type,
+            type: TxVariants.card,
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/Card/Card.tsx:68]: Do we still need the `TxVariant.card` ? What is it used for? I expected that we would need only `TxVariant.scheme`

File: packages/lib/src/components/Ach/components/AchInput/AchInput.tsx (hunk 1 above)
Comment (sponglord) [reply packages/lib/src/components/Ach/components/AchInput/AchInput.tsx]: I think there was a merge error. It should be fixed now

File: packages/lib/src/components/Card/Card.tsx (hunk 2 above)
Comment (sponglord) [reply packages/lib/src/components/Card/Card.tsx:68]: `this.props.type` (with value `TxVariant.card`) is an internal property that propagates down to the securedFields to tell them what "type" they are.
All the securedFields code is built around being a "card"

Comment (ribeiroguilherme) [reply packages/lib/src/components/Card/Card.tsx:68]: Ah, ok.. seems to be a massive change then 😅

**Pull request merged**